*enclosing-call-site.js:16
      throw new Error('an error!')
                ^

Error: an error!
    at functionD (*enclosing-call-site.js:16:17)
    at functionC (*enclosing-call-site.js:10:3)
    at functionB (*enclosing-call-site.js:6:3)
    at functionA (*enclosing-call-site.js:2:3)
    at Object.<anonymous> (*enclosing-call-site.js:24:3)
